src/compounds/Item/CardBody.jsx

```diff
@@ -4,7 +4,7 @@ import { Button, Card, Collapse } from 'react-bootstrap'
 import NextLink from '../../components/NextLink/NextLink'
 import LinkedButton from '../LinkedButton/LinkedButton'
 
-const CardBody = ({ buttonLink, buttonProps, item, markdownDescriptionTruncated, markdownDescriptionExtended, orientation, titleLink, withButtonLink, withTitleLink }) => {
+const CardBody = ({ buttonLink, buttonProps, item, markdownDescriptionTruncated, markdownDescriptionExtended, truncatedLength, orientation, titleLink, withButtonLink, withTitleLink }) => {
```

Comment:
what do you think about changing this variable name to `truncateAt`? feels a little more semantic.

also, can you move this to after "titleLink"? the variables are alphabetized.

Reply:
Summer called it cutOffIndex. Do you think it makes sense to use that?

Reply:
I feel like `truncateAt` is even more semantic than `cutOffIndex`...but I'm fine with either.